maat-court-data-api/src/test/java/gov/uk/courtdata/assessment/validator/FinancialAssessmentValidationProcessorTest.java

```diff
@@ -76,4 +80,11 @@ public void testFinancialAssessmentValidationProcessor_whenRequiredFieldsPresent
         assertThat(financialAssessmentValidationProcessor.validate(assessment)).isEqualTo(Optional.empty());
     }
 
+    @Test
+    public void testUpdateAppDateCompletedValidator_whenRequiredFieldsPresent_thenValidationPasses() {
+        UpdateAppDateCompleted updateAppDateCompleted = TestModelDataBuilder.getUpdateAppDateCompleted();
```

Comment:
I think just checking that the correct validator was called with verify() is enough.

Reply:
Fixed now